sabnzbd/config.py

```diff
@@ -93,6 +93,7 @@ def get_dict(self, safe=False):
 
     def set_dict(self, dict):
         """ Set value based on dictionary """
+        # TODO: Review if `dict` is actually correct or typo
```

Comment:
Why do you think this is wrong? It seems to be a prototype to be overwritten by the inherited classes that use it to easily submit config settings.